bool require_key_phrase
bool overwrite_existing
int max_speech_length
int silence_timeout